src/Login/LoginPassword.java

```diff
@@ -0,0 +1,26 @@
+package Login;
+
+import java.util.Scanner;
+public class LoginPassword {
+    public void loginpassword() {
+        String Username = "Oleksii";
+        String Password = "371";
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Enter Username : ");
+        String username = sc.next();
+        System.out.println("Enter Password : ");
+        String password = sc.next();
+        if (username.equals(Username) && password.equals(Password)) {
+            System.out.println("Access Granted! Welcome!");
+            System.out.println("Enter number from 1 to 5: ");
+        } else if (username.equals(Username)) {
+            System.out.println("Invalid Password!");
```

Comment:
no need to segregate response with Invalid Password/Username, provide one generic error if either Username or Password is wrong. Add System.exit(1) method in the end